### ISSUE TITLE
Create 32-bit Linux build

### ISSUE THREAD

**Issue by dominictarr** [feross/webtorrent-desktop#335]
I get this error...
```
~/Downloads/WebTorrent-linux-x64$ ./WebTorrent 
bash: ./WebTorrent: cannot execute binary file: Exec format error
```

maybe you also need a 32 bit linux?

**Comment by grunjol** [comment]
You are trying to running a 64 bit program in a 32 bit OS.

**Comment by feross** [comment]
@dominictarr Does your processor not support 64-bit? Or did you just pick the 32-bit distribution?

We can make a 32-bit build if necessary.

**Comment by dominictarr** [comment]
@feross I can't remember.

following this: https://www.linuxquestions.org/questions/linux-general-1/how-to-check-linux-kernel-is-32-bit-or-64-bit-612352/

```
> uname -m
i686
> uname -ar
Linux x201 4.2.0-22-generic #27-Ubuntu SMP Thu Dec 17 22:57:22 UTC 2015 i686 i686 i686 GNU/Linux
> getconf LONG_BIT
32
```
looks like I'm 32 bit

**Comment by grunjol** [comment]
In 64bit platform you will see
```
uname -m
x86_64

```

**Comment by feross** [comment]
@dominictarr Can you try running `npm install webtorrent-desktop -g` and then `webtorrent-desktop`?

**Comment by dominictarr** [comment]
@feross yup, npm install worked.

**Comment by dominictarr** [comment]
npm install is only the easy way to install if you are a node developer

because os package managers often don't keep their node versions very up to date,
so you probably need to use nvm or something, and that is a pain.

**Comment by ungoldman** [comment]
It would not be too difficult to reconfigure the package script to produce both 64 and 32 bit distributions. We did it for friends. If it gets the application to a wider audience and doesn't complicate anything other than packaging I vote we Just Do It

**Comment by yciabaud** [comment]
Another user on chat was stuck on 32bit OS, he did not have node installed and I think he gave up trying webtorrent desktop because of this.
A 32bit package makes sense.

**Comment by feross** [comment]
@yciabaud Was he a linux user too?

**Comment by oseido** [comment]
Same issue on a ubuntu 14.04 32 bit version. Can you build a 32 bit version of the WebTorrent Desktop ?

Shell Output:
-
Desktop WebTorrent: 1: WebTorrent: ELF: not found
WebTorrent: 1: WebTorrent: : not found
WebTorrent: 4: WebTorrent: @!@@@@@00pp@p@@@���� �������'��- �[��[�[��@�@DD�����P�td��P�P����Q�tR�td������%%/lib64/ld-linux-x86-64.so.2GNUGNU��>�t@B��J,y,S�
                                                                                     "	E��"�F�0$����t
                                                                                                          �@8�	  @\��@�P�"4A@���     A��H`A&�"Hs�`C�5P@K����@�
                                         A "LL�
H�(�$4 ���
��1X��@�i�����H5: not found
WebTorrent: 1: WebTorrent: cannot open @�*@
                                            ��@: No such file
WebTorrent: 1: WebTorrent: �B: not found
WebTorrent: 1: WebTorrent: �: not found
WebTorrent: 5: WebTorrent:8�!D�Y �BF
�                                    �xL��1�'CR�	P��
  ��
       ��D#(
                 �@C�1�E#��$
                                  � �	Ȅ�AJ�� H�P8�A��:�
                                                               P@��a�@�B@&�% 	�p��
                                                                                               0Bb�D��A8�E$!A: not found
WebTorrent: 6: WebTorrent: �
                            ��: not found
WebTorrent: 19: WebTorrent: Syntax error: ")" unexpected

**Comment by yciabaud** [comment]
@feross yes, Linux Ubuntu 14.04 32bit on a 64bit cpu

**Comment by feross** [comment]
Okay, I'm okay if we do this now. PR welcome :)

**Comment by feross** [comment]
This is completed now, and will be available in the next release v0.4.0.
